packages/cfd/src/Containers/cfd-password-modal.tsx

```diff
@@ -526,7 +528,12 @@ const CFDPasswordForm = ({
                                         i18n_default_text='Enter your {{platform}} password to add a {{platform_name}} {{account}} {{jurisdiction_shortcode}} account.'
                                         values={{
                                             platform: getCFDPlatformLabel(platform),
-                                            platform_name: platform === CFD_PLATFORMS.MT5 ? 'MT5' : 'Deriv X',
+                                            platform_name:
+                                                platform === CFD_PLATFORMS.MT5
+                                                    ? 'MT5'
+                                                    : CFD_PLATFORMS.DXTRADE
+                                                    ? 'Deriv X'
+                                                    : 'Deriv Ez',
```

Comment:
@shontzu-deriv, is it possible to extract this out to a function? I think that generally, nested ternary operators can be confusing.
```suggestion
                                            platform_name:
                                                getPlatformName(platform) 
```
Then:

```
const getPlatformName = (platform) => {
if(platform === CFD_PLATFORMS.MT5){
                                                    return  'MT5';
                                                    else if(platform === CFD_PLATFORMS.DXTRADE){
                                                    return 'Deriv X';
} esle{
                                                    return 'Deriv Ez';
}
}
```

Reply:
Noted with thanks! Will discuss with @hirad-deriv and refactor the code to prevent merge conflicts

Reply:
I have already created the function needed for this matter. After that cTrader is merged, you can use that function.